infra/label_evaluator_step_functions/TRACE_SPEC.md

```
@@ -0,0 +1,531 @@
+    "ResultPath": "$.patterns_result",
+    "Next": "ReturnPatternResult"
+  }
+}
+```
+
+---
+
+## Phase 4: Define "Jobs" for EMR Analytics
+
+### 4.1 Job Types
+| Job Type | Trace Name | Scope | Children |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)